src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler/Deprecation.php

```diff
@@ -280,12 +283,12 @@ private static function getVendors()
         return self::$vendors;
     }
 
-    private static function getSourcePathsFromPrefixes(array $prefixesByNamespace)
+    private static function addSourcePathsFromPrefixes(array $prefixesByNamespace, array &$paths)
```

Comment:
can you remove the reference here please? returning an array should make it, isn't it?

Reply:
Yes, using reference is not required; updated :)